### ISSUE TITLE
Individual styles, key bindings for multiple trays

### ISSUE THREAD

**Issue by coreymwamba** [joewing/jwm#282]
This is a collection of questions/feature requests; but they all involve trays. I wasn't certain whether to open separate issues. 

1) Is it possible to set a key binding to toggle an auto-hidden tray *instead* of moving the mouse cursor to the edge? Currently I have a key binding to bring up a tray with a task list (see screenshot), but it is also activated when I move the mouse to the top edge of the screen. I'd just like it to be toggled with the key binding instead.
![jwm_trays](https://cloud.githubusercontent.com/assets/2705306/12870949/ec91abda-cd55-11e5-90fa-83dd9b4715cf.png)

2) Is it possible to set colour/font/size/transparency options for each tray, rather than just the global TrayStyle?

**Comment by Israel-** [comment]
Hi, the first one is already implemented
Here is an example:
`<Key mask="4" key="t">showtray</Key>`
For the second one, I do not think this is possible currently, but i understand why you want to, especially if you have a Tray that is more like a dock application.

**Comment by technosaurus** [comment]
I mentioned the individual tray and menu styles in https://github.com/joewing/jwm/issues/247

Some of that was also to implement styles as properties for a variety of reasons, but from the user perspective the biggest one is the ability to have unique trays and menus for varied actions.

**Comment by joewing** [comment]
Support for multiple tray styles is something I’ve been considering (see issue #39).

Currently, there isn’t a way to have a tray key binding without also having the mouse show the tray.  I could see that being useful though.

**Comment by coreymwamba** [comment]
> Currently, there isn’t a way to have a tray key binding without also
> having the mouse show the tray.  I could see that being useful though.

Indeed, this is exactly what I was thinking. Sadly, "showtray" only
works if the cursor is in same area as the tray itself (which I can see
would be useful in some cases, but just not the one I need).
